### PR TITLE
Loan program content + layout fixes

### PR DESCRIPTION
**Pull request by jrubenoff**
### Content changes
- Rewrote program info for clarity and readability
- Added that EIDL isn't just for small businesses, but also nonprofits and veteran's orgs
- Updated the EIDL program CTA to take you directly to the application, not the info page
- Moved the "EIDL advance" into the "EIDL" card, since you can't get an advance without applying for an EIDL first
- Updated the link to apply for 7(a) loans
- Added the specific cutoff date for Small Business Debt Relief programs
- Added a link to check if your 500+ employee business qualifies for these programs

### Markup changes
- Move loan program content to a `<dl>`
- Switched the markup to use Bootstrap's `.card` component, since it looks a bit nicer
- Changed loan program CTAs to buttons, adding the full URLs below them for the print stylesheet
- [unrelated] Changed the button font size to `100%` to avoid multi-line buttons on small screens

### CODE REVIEW

**Comment by jrubenoff** [comment]
@jasimon @ahupp Feel free to assign yourself to this instead if you have time to review it soon...

File: docs/index.html [hunk 1]
```diff
@@ -148,150 +148,195 @@ <h3>Based upon your feedback, you may be eligible for the following <strong>fede
...
+                  <dd class="col-sm-9">
+                    <ul>
+                      <li>Businesses with under 500 employees</li>
+                      <li>Private non-profit organizations</li>
+                      <li>501(c)(19) veterans’ organizations</li>
+                      <li>Some businesses with 500+ employees.<br/><a href="https://www.sba.gov/size-standards/" target="_blank">Check to see if your business qualifies.</a></li>
+                    </ul>
+                  </dd>
+
+                  <dt class="col-sm-3">What can I use it&nbsp;for?</dt>
```
Comment (ahupp) [inline docs/index.html:170]: Just curious when you'd choose to us &nbsp?

Comment (ahupp) [reply docs/index.html:170]: As in, I know what it does, but why in particular here?

Comment (jrubenoff) [reply docs/index.html:170]: The word "for" was wrapping to a new line, so this prevents typographic orphans. (If we were using anything other than raw HTML, I'd add a helper function for this...)

**Pull request merged**